packages/exporter-trace-otlp-grpc/README.md

```diff
@@ -107,6 +107,29 @@ provider.register();
 
 Note, that this will only work if TLS is also configured on the server.
 
+## Exporter Timeout Configuration
```

Comment:
Documentation how to configure timeout in GRPC exporter is added here but I miss any changes for the actual implementation. Is this working already but just not documented?